src/screens/LoginScreen.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { shape } from 'prop-types';
+import {
+  SafeAreaView,
+  StyleSheet,
+  ScrollView,
+  View,
+  Text,
+  Image,
+  Button,
+} from 'react-native';
+import Login from '../components/Login/Login';
+import logo from '../../assets/images/vermonster-logo.png';
+
+const LoginScreen = ({ navigation }) => (
+  <SafeAreaView style={styles.safeAreaView}>
```

Comment:
The following documentation suggests that`SafeAreaView` might be iOS only -- is this accurate?
https://reactnative.dev/docs/safeareaview

>The purpose of SafeAreaView is to render content within the safe area boundaries of a device. It is currently only applicable to iOS devices with iOS version 11 or later.

Reply:
Interestingly, android does not have an elegant solution for this. Instead you need to import `StatusBar` and measure the height.